packages/lisk-transactions/src/4_multisignature_transaction.ts

```diff
@@ -331,4 +332,65 @@ export class MultisignatureTransaction extends BaseTransaction {
 
 		return [];
 	}
+
+	public addMultisignature(
+		store: StateStore,
+		signatureObject: SignatureObject,
+	): TransactionResponse {
+		// Get the account
+		const account = store.account.get(this.senderId);
+		// Check if signature is not already there
+		if (this.signatures.includes(signatureObject.signature)) {
+			return createResponse(this.id, [
+				new TransactionError(
+					`Signature '${
+						signatureObject.signature
+					}' already present in transaction.`,
+					this.id,
+				),
+			]);
+		}
+
+		// Validate signature key belongs to pending multisig registration transaction
+		// tslint:disable-next-line
+		const keysgroup = (this as any).asset.multisignature.keysgroup.map(
```

Comment:
why do we need `this as any`? prefer not to use any

Reply:
Was needed in base transaction but not here. Removing.